### PR TITLE
Map some things mostly related to commands

### PR DESCRIPTION
**Pull request by williambl**
- `class_38` -> `Command` (or maybe `CommandInfo` would be better?)
 - `class_11` -> `ServerPlayerPacketHandler` (I wasn't too sure about this one. But it's a PacketHandler that's linked to a ServerPlayer...)
 - `class_426` -> `CommandManager`
 - `class_39` -> `CommandSource`

### CODE REVIEW

**Comment by valoeghese** [comment]
Can you bump the build number in build.gradle

**Comment by williambl** [comment]
Done.

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 1]
```diff
@@ -1,16 +1,22 @@
 CLASS net/minecraft/server/MinecraftServer
-	CLASS class_643 ServerThread
-	CLASS class_645 ConsoleInputThread
```
Comment (williambl) [inline mappings/net/minecraft/server/MinecraftServer.mapping:3]: Wait, what's happening here? Did I accidentally unmap these?

Comment (valoeghese) [reply mappings/net/minecraft/server/MinecraftServer.mapping:3]: Enigma moved them to the bottom I think

File: mappings/net/minecraft/server/MinecraftServer.mapping [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 	METHOD main ([Ljava/lang/String;)V
 		ARG 0 args
 	METHOD method_2160 getFile (Ljava/lang/String;)Ljava/io/File;
+	METHOD method_2162 queueCommand (Ljava/lang/String;Lnet/minecraft/class_39;)V
+		ARG 1 commandString
+		ARG 2 source
+	METHOD method_2164 processQueuedCommands ()V
 	METHOD method_2166 start ()Z
 	METHOD method_2167 logWarning (Ljava/lang/String;)V
+	CLASS class_643 ServerThread
```
Comment (valoeghese) [inline mappings/net/minecraft/server/MinecraftServer.mapping:20]: They’re here

**Pull request merged**